tests/featureCompiler_test.py

```diff
@@ -100,6 +100,18 @@ def test_include_dir(self, FontClass, tmp_path, caplog):
 
         assert "# hello world" in str(fea)
 
+    def test_include_dir_cwd(self, FontClass, tmp_path, monkeypatch):
+        (tmp_path / "test.fea").write_text("# hello world", encoding="utf-8")
+        ufo = FontClass()
+        ufo.features.text = "include(test.fea)"
+
+        with monkeypatch.context() as context:
```

Comment:
elsewhere in this same file we use a contextmanager called `pushd` defined in testSupport.py, that does the same thing, you may want to use that here as well for consistency maybe if you like

Reply:
i'm ok with that as long as we use the same method for both